src/components/HomePage/Header.tsx

```diff
@@ -144,6 +153,22 @@ const Header: React.FC = () => {
         <HeaderContent />
         <HeaderImage />
       </div>
+      <motion.button
```

Comment:
don't use button in `src/components/HomePage/Header.tsx`. You need to use after https://github.com/CodeHarborHub/codeharborhub/blob/8ab1c372213a93492013b482264998b67fd8b895/src/pages/index.tsx#L143